src/main/scala/uk/gov/homeoffice/drt/applicative/ArrivalFlights.scala

```diff
@@ -1,24 +1,25 @@
 package uk.gov.homeoffice.drt.applicative
 
-import cats.Applicative
 import cats.syntax.all._
+import cats.{Applicative, Monad}
 import uk.gov.homeoffice.drt.model.{ArrivalTableDataIndex, Arrivals, FlightsRequest}
-import uk.gov.homeoffice.drt.service.ArrivalService
+import uk.gov.homeoffice.drt.service.FlightScheduledService
 
 
 trait ArrivalFlights[F[_]] {
-  def flights(n: FlightsRequest): F[Arrivals]
+  def flights(n: FlightsRequest)(implicit monad: Monad[F]): F[Arrivals]
 }
 
 object ArrivalFlights {
   implicit def apply[F[_]](implicit ev: ArrivalFlights[F]): ArrivalFlights[F] = ev
 
-  def impl[F[_] : Applicative](arrivalsService: ArrivalService[F]): ArrivalFlights[F] = new ArrivalFlights[F] {
-    def flights(n: FlightsRequest): F[Arrivals] = {
-      val a: F[List[ArrivalTableDataIndex]] = arrivalsService.getFlightsDetail(n)
-      arrivalsService.transformArrivals(a).map(Arrivals(_))
+  def impl[F[_] : Applicative](arrivalsService: FlightScheduledService[F]): ArrivalFlights[F] = new ArrivalFlights[F] {
+    def flights(n: FlightsRequest)(implicit monad: Monad[F]): F[Arrivals] = {
+      val arrivalFlights: F[List[ArrivalTableDataIndex]] = arrivalsService.getFlightsDetail(n)
```

Comment:
`arrival` is probably better than `arrivalFlights` seeing as it means the same thing.